torch_glow/tests/nodes/max_test.py

```diff
@@ -0,0 +1,19 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import torch
+import torch_glow
+from tests.utils import jitVsGlow
+
+
+# Test of the PyTorch max Node on Glow.
+
```

Comment:
too much spacing here

Reply:
Do you dare defy `autopep8`?

Reply:
Is that really what python comments are supposed to look like? Seems too weird. Are we supposed to use the """ inside the function style or something (https://www.python.org/dev/peps/pep-0008/#id33)?

Reply:
Yep, docstrings in Python are usually written after the function prototype:
```
def max_test(...):
"""Tests max node"""
```